app/tests/unit/containers/Incentives/IncentiveCalculatorContainer.test.tsx

```diff
@@ -27,6 +57,15 @@ describe('IncentiveCalculator', () => {
     expect(r).toMatchSnapshot();
   });
 
+  it('includes an aggregate row with multiple products', async () => {
+    const r = shallow(
+      <IncentiveCalculator
+        applicationRevision={applicationRevisionMultiProduct}
+      />
+    );
+    expect(r).toMatchSnapshot();
```

Comment:
the snapshot matches are more of a "catch all" which would show us unexpected changes. Our unit tests should have some stronger assertions to complement them: in this case checking that the displayed totals match the expected value